### ISSUE TITLE
pSortableColumn directive uses [aria-sort], but it should be [attr.aria-sort] according to angular docs

### ISSUE THREAD

**Issue by ZacharyWigfield** [primefaces/primeng#18571]
### Describe the bug

When unit testing with jasmine/karma on components that use a primeng table with <th> using the pSortableColumn directive, karma shows the following error:

Error: NG0303: Can't bind to 'aria-sort' since it isn't a known property of 'th' (used in the 'DisasterTableComponent2' component template).

Looking into the source code, pSortableColumn directive uses [aria-sort], but according to angular documentation it should be [attr.aria-sort]. I believe my testing issue is related, but either way it should probably be amended. 

To replicate my error, see attached stackblitz and running ng test should show the issue. 

### Pull Request Link

https://github.com/primefaces/primeng/pull/18570

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/github-uxazmatq?file=src%2Fapp%2Fp-table-single-column-sort%2Fp-table-single-column-sort.spec.ts

### Environment

My environment is identical to the stackblitz, using angular and primeng version 20. 

### Angular version

20.0.6

### PrimeNG version

v20

### Steps to reproduce the behavior

run ng test 

### Expected behavior

jasmine/karma should be able to bind to aria-sort

**Comment by mertsincan** [comment]
Fixed in https://github.com/primefaces/primeng/pull/18570
